tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/prj.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_CPLUSPLUS=y
+CONFIG_NEWLIB_LIBC=y
+CONFIG_LOG=n
```

Comment:
This file adds `CONFIG_LOG=n` and `CONFIG_I2S_LITEX_DATA_BIG_ENDIAN=y`. The other lines are already in our project. I suspect that I2S configuration is necessary but I am not sure what `CONFIG_LOG` does.